src/models/userProfile.js

```diff
@@ -83,6 +83,8 @@ const userProfileSchema = new Schema({
     education: { type: Number, default: 0 },
     society: { type: Number, default: 0 },
     energy: { type: Number, default: 0 },
+    economics: { type: Number, default: 0 },
```

Comment:
Fulfilled "Please add categories for: Total Stewardship Hours and Total Economics Hours."  Good job.